scripts/ssl-manager.js

```diff
@@ -587,12 +629,14 @@ function SSLManager(config) {
             nodes,
             resp;
 
-        if ((!id && !group) || !nodeManager.isBalancerLayer(group)) {
-            resp = nodeManager.getEntryPointGroup();
-            if (resp.result != 0) return resp;
+        if (!config.webroot) {
+            if ((!id && !group) || !nodeManager.isBalancerLayer(group)) {
+                resp = nodeManager.getEntryPointGroup();
```

Comment:
add webroot condition to getEntryPointGroup

Reply:
done